app/assets/v2/js/pages/bounty_details2.js

```diff
@@ -392,12 +400,17 @@ Vue.mixin({
     },
     getTenant: function(token_name, web3_type) {
       let tenant;
+      let vm = this;
 
       if (web3_type == 'manual') {
         tenant = 'OTHERS';
         return tenant;
       }
 
+      if (vm.bounty.web3_type == 'nervos_ext') {
+        vm.canChangeFunderAddress = true;
+      }
```

Comment:
^ could we make this into an array ['nervos_ext'] cause we have one more chain coming in which would want this feature.